packages/cli/src/settings.ts

```diff
@@ -37,9 +37,14 @@ export type CliSettings = {
   ipfsTimeout?: number;
 
   /**
-   * the url of the IPFS endpoint to use as a storage base. defaults to localhost IPFS
+   * the url of the IPFS endpoint to use as a storage base, only for reading data
    */
-  ipfsUrl?: string;
```

Comment:
I thought the idea is we leave this and deprecate it? I think it'd be a fallback for readIpfsUrl?

Reply:
really like to avoid keeping it around if we are taking this direction